src/FastifyZod.ts

```diff
@@ -6,6 +6,11 @@ import {
 } from "fastify";
 import fastifySwagger, { FastifyDynamicSwaggerOptions } from "@fastify/swagger";
 import * as yaml from "js-yaml";
+import {
+  FastifyTypeProviderDefault,
+  ResolveFastifyReplyReturnType,
+  // eslint-disable-next-line import/no-unresolved
+} from "fastify/types/type-provider";
```

Comment:
I was not able to figure out why this module is marked as unresolved, the imports work fine.. Happy to apply any changes if someone knows how to fix this...